mappings/net/minecraft/block/Block.mapping

```diff
@@ -120,93 +136,155 @@ CLASS net/minecraft/class_17 net/minecraft/block/Block
 	METHOD <init> (ILnet/minecraft/class_15;)V
 		ARG 1 id
 		ARG 2 material
-	METHOD method_1560 (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;)V
+	METHOD method_1560 onSteppedOn (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 		ARG 5 entity
-	METHOD method_1562 (Lnet/minecraft/class_18;IIILnet/minecraft/class_25;Ljava/util/ArrayList;)V
+	METHOD method_1561 canPlaceAt (Lnet/minecraft/class_18;IIII)Z
+		ARG 1 world
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 side
+	METHOD method_1562 addIntersectingBoundingBox (Lnet/minecraft/class_18;IIILnet/minecraft/class_25;Ljava/util/ArrayList;)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 		ARG 5 box
-	METHOD method_1563 (Lnet/minecraft/class_18;IIIII)V
+		ARG 6 boundingBoxes
+	METHOD method_1563 onBlockAction (Lnet/minecraft/class_18;IIIII)V
+		ARG 1 world
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 data1
+		ARG 6 data2
+	METHOD method_1564 raycast (Lnet/minecraft/class_18;IIILnet/minecraft/class_26;Lnet/minecraft/class_26;)Lnet/minecraft/class_27;
 		ARG 1 world
-	METHOD method_1564 (Lnet/minecraft/class_18;IIILnet/minecraft/class_26;Lnet/minecraft/class_26;)Lnet/minecraft/class_27;
-		ARG 1 word
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1566 (Lnet/minecraft/class_18;III)Z
+		ARG 5 startPos
+		ARG 6 endPos
+	METHOD method_1565 getTickRate ()I
+	METHOD method_1566 canGrow (Lnet/minecraft/class_18;III)Z
 		ARG 1 world
-	METHOD method_1567 (Lnet/minecraft/class_18;III)Z
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_1567 canPlaceAt (Lnet/minecraft/class_18;III)Z
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1568 (Lnet/minecraft/class_14;IIII)Z
+	METHOD method_1568 isEmittingRedstonePower (Lnet/minecraft/class_14;IIII)Z
 		ARG 1 blockView
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1570 (Lnet/minecraft/class_18;IIII)Z
+		ARG 5 meta
+	METHOD method_1569 canEmitRedstonePower ()Z
+	METHOD method_1570 canTransferPowerInDirection (Lnet/minecraft/class_18;IIII)Z
 		ARG 1 world
-	METHOD method_1572 (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;Lnet/minecraft/class_26;)V
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 direction
+	METHOD method_1571 hasCollision (IZ)Z
+		ARG 1 meta
+	METHOD method_1572 onEntityCollision (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;Lnet/minecraft/class_26;)V
```

Comment:
Should be `applyVelocity`, as it's similar in usage to `Fluid#getVelocity` in modern, but applies the velocity to the vector manually instead of returning a modifier vector.